modules/react-components/src/layouts/page.tsx

```diff
@@ -21,6 +21,7 @@ import classNames from "classnames";
 import React, { FunctionComponent, PropsWithChildren, ReactElement } from "react";
 import { Divider } from "semantic-ui-react";
 import { PageHeader, PageHeaderPropsInterface } from "../components";
+import {Helmet} from "react-helmet"
```

Comment:
Check if `ESLint` works.

```suggestion
import { Helmet } from "react-helmet"
```

Reply:
will check